src/plugins/discover/public/application/main/components/sidebar/lib/get_field_list.ts

```diff
@@ -74,6 +74,7 @@ export function getTextBasedQueryFieldList(
         type: column.meta?.type ?? 'unknown',
         searchable: false,
         aggregatable: false,
+        empty: Boolean(column?.isNull),
```

Comment:
Can we use `isNull` instead of `empty`? It would be more readable and consistent.

Reply:
Sure, [done](https://github.com/elastic/kibana/pull/174585/commits/04210d6b4f69545730259eccf824213e6885dc2c)